jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/CawsConnectorViewPanels.kt

```diff
@@ -136,6 +141,10 @@ fun cawsWizard(lifetime: Lifetime, settings: CawsSettings = CawsSettings()) = Mu
                         error("Not implemented")
                     }
 
+                    if (context.is3P) {
```

Comment:
Is this check here required? Won't this always be false initially?

Reply:
Check is required to switch branchType to Existing, when 3P repo is selected